src/scenes/app/reducers/actions.js

```diff
@@ -0,0 +1,98 @@
+import actionTypes from './actionTypes';
+import ToDoController from '../../../networking/controllers/ToDoController';
+import strings from '../../../localization/en/strings';
+import { deserializeItems, deserializeItem } from '../../../helpers/serializer';
+
```

Comment:
I suggest adding the actions & reducers shared by multiple components to a separate directory `globalReducers` within `/src`